### ISSUE TITLE
Unable to link github repository behind proxy

### ISSUE THREAD

**Issue by charly71** [kactus-io/kactus#127]
## Description

In our company we are unable to access to Github repositories trough command line without setting a proxy. 

## Version

* Kactus:
0.3.19

* Operating system:
macOS 10.14.3

## Steps to Reproduce

1. Create an empty repository on Github
2. Open Kactus
3. Click on "Select Repository"
4. Cut & paste the repository url
5. Try to connect...
### Expected Behavior

I expect to connect correctly to my remote repository
### Actual Behavior

The system hang and after seconds stops with this error:

Cloning into '/Volumes/g08/02 - Documentazione di Progetto/2019/UX deliverable/Test/provakactus'...
fatal: unable to access 'https://github.com/charly71/provakactus.git/': Failed to connect to github.com port 443: Operation timed out

**Comment by mathieudutour** [comment]
Currently, you still need to tell Git about your proxy settings:
```
git config --global http.proxy http://proxyuser:proxypwd@proxy.server.com:8080
```
Where `proxyuser` and `proxypwd` are your credentials, and `proxy.server.com:8080` is the server name and port you connect to.

I'll see if I can make it easier but it might take a bit

**Comment by mathieudutour** [comment]
fixed in 0.3.27
